pagegraph/src/types.rs

```diff
@@ -75,6 +75,7 @@ impl From<&str> for RequestType {
         match v {
             "Image" => Self::Image,
             "ScriptClassic" => Self::ScriptClassic,
+            "Script" => Self::ScriptClassic,
```

Comment:
I'm not sure why we have ScriptClassic in the first place.

Reply:
Should we rename `ScriptClassic` to `Script` then? We control 100% of the format, I don't think it makes sense to keep both around if we're gonna alias them like this here.

Reply:
Ya that makes sense

Reply:
I wasn't sure why it existed in the first place

Reply:
I mean, because that's what the graphml file looked like I'm sure. But that seems to have changed.